internal/jobs/rename_instances_aws_job.go

```diff
@@ -0,0 +1,47 @@
+package jobs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/lzap/dejq"
+)
+
+type RenameInstanceJobArgs struct {
+	AccountID     int64             `json:"account_id"`
+	ReservationID int64             `json:"reservation_id"`
+	Detail        *models.AWSDetail `json:"detail"`
+}
+
+// Unmarshall arguments and handle error
+func HandleRenameInstance(ctx context.Context, job dejq.Job) error {
+	args := NoopJobArgs{}
```

Comment:
Why do you use Noop args?

Reply:
Copy-pasting, this is a bug actually :-) Weird it worked the last time I tried this.

Reply:
Oh wait, this is the empty job, I haven't tested it yet. Yup, copy and paste problem, fixing.